backend/apps/accounts/views.py

```
@@ -860,59 +904,62 @@ def get(self, request):
             status=status.HTTP_400_BAD_REQUEST,
         )
 
+
 class AvatarUploadView(APIView):
     permission_classes = [IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)